command/agent.go

```diff
@@ -219,6 +219,12 @@ func (c *AgentCommand) Run(args []string) int {
 			Default: "https://127.0.0.1:8200",
 			EnvVar:  api.EnvVaultAddress,
 		})
+		c.setStringFlag(f, config.Vault.Namespace, &StringVar{
```

Comment:
If we end up implementing this, we should also print in during agent's startup (probably by appending to `infoKeys `).